src/server_manager/model/account_manager.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {LocalStorageRepository} from '../infrastructure/repository';
+
+import {Account, AccountModelFactory, Data} from './account';
+import * as cloud_provider from './cloud_provider';
+
+export class AccountManager {
```

Comment:
Perhaps we need a concept "ConnectedAccounts" repository, with methods like `connectDigitalOceanAccount(DOCredentials)`, `connectGcpAccount(GcpCredentials)` ...

You can have a `listAccounts() CloudAccount[]` method for the side bar, and each CloudAccount can have a listServers() method.

You shouldn't be exposing "data". That's not a product concept. A model concept shouldn't depend on LocalStorage either.